### PR TITLE
remove npa nxx reference

### PR DESCRIPTION
**Pull request by InfAmelia**

### CODE REVIEW

File: source/api_documentation/network_integration/promo_numbers/index.rst [hunk 1]
```diff
@@ -174,5 +174,3 @@ Endpoint:
    :path: /promo_numbers/&lt;promo_number&gt;
    :description: Delete a Campaign Promo Number
    :page: delete_promo_number
-
-
```
Comment (peterwilson) [inline source/api_documentation/network_integration/promo_numbers/index.rst]: White space at the end of the file is important, make sure this isn't deleted!

**Review by peterwilson — CHANGES_REQUESTED** [review]
Whitespace is necessary at the end of the file

File: source/api_documentation/network_integration/promo_numbers/index.rst (hunk 1 above)
Comment (InfAmelia) [reply source/api_documentation/network_integration/promo_numbers/index.rst]: developer-docs____invoca_developer-docs__-_____source_api_documentation_network_integration_promo_numbers_index_rst__developer-docs_

 I still see the newlines there when I open this up locally, I'm not sure why this is showing up

**Comment by InfAmelia** [comment]
Newlines ensured at EOF

**Review by peterwilson — APPROVED** [review]

**Pull request merged**